src/Controls/src/Core/Platform/Windows/CollectionView/FormsGridView.cs

```diff
@@ -152,10 +146,10 @@ protected override void OnApplyTemplate()
 
 		protected override global::Windows.Foundation.Size ArrangeOverride(global::Windows.Foundation.Size finalSize)
 		{
-			if (_formsEmptyView != null)
-			{
-				_formsEmptyView.Layout(new Rect(0, 0, finalSize.Width, finalSize.Height));
-			}
+			_formsEmptyView?.Layout(new Rect(0, 0, finalSize.Width, finalSize.Height));
+
+			if (_wrapGrid is null)
```

Comment:
not sure to what extent this matters, but it looks like you can subscribe to ChoosingItemContainer and call FindItemsWrapGrid from there. 

It just feels a bit weird to set the size information of the WrappedGrid inside the ArrangeOverride call.

Thoughts on just calling it from there?

```C#
public FormsGridView()
{
	// Using the full style for this control, because for some reason on 16299 we can't set the ControlTemplate
	// (it just fails silently saying it can't find the resource key)
	DefaultStyleKey = typeof(FormsGridView);

	RegisterPropertyChangedCallback(ItemsPanelProperty, ItemsPanelChanged);

	this.ChoosingItemContainer += FormsGridView_ChoosingItemContainer;
}

private void FormsGridView_ChoosingItemContainer(ListViewBase sender, ChoosingItemContainerEventArgs args)
{
	if (_wrapGrid is not null)
		FindItemsWrapGrid();
}
```

And then even for good measure 

```C#
protected override global::Windows.Foundation.Size MeasureOverride(global::Windows.Foundation.Size availableSize)
{
	var result =  base.MeasureOverride(availableSize); // it looks like the grid comes into existence here.

	FindItemsWrapGrid();

	return result;
}
```


I also wonder if we should null at the grid and unsubscribe when the orientation is changed.

The Orientation applies a different template so the wrapgrid is going to change. 
But maybe this GridView is recreated when the orientation gets changed?
```C#
		public Orientation Orientation
		{
			get => _orientation;
			set
			{
				_orientation = value;
				if (_orientation == Orientation.Horizontal)
				{
					ItemsPanel = (ItemsPanelTemplate)UWPApp.Current.Resources["HorizontalGridItemsPanel"];
					ScrollViewer.SetHorizontalScrollMode(this, WScrollMode.Auto);
					ScrollViewer.SetHorizontalScrollBarVisibility(this, UWPControls.ScrollBarVisibility.Auto);
				}
				else
				{
					ItemsPanel = (ItemsPanelTemplate)UWPApp.Current.Resources["VerticalGridItemsPanel"];
				}
			}
		}
```